contracts/bonding/BondingManager.sol

```diff
@@ -841,16 +888,35 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
             earningsPool.setStake(t.earningsPoolPerRound[lastUpdateRound].totalStake);
         }
 
+        if (treasuryBalanceCeiling > 0) {
+            uint256 treasuryBalance = livepeerToken().balanceOf(treasury());
+            if (treasuryBalance >= treasuryBalanceCeiling && nextRoundTreasuryRewardCutRate > 0) {
+                // halt treasury contributions until the cut rate param is updated again
+                _setTreasuryRewardCutRate(0);
```

Comment:
"reposting" here cause github hid the comment in the commit.
https://github.com/livepeer/protocol/commit/ead1edb572319df53dbfa06c3466edb4ca8c0575#r124819767

Reply:
Left my comment in that link.